### ISSUE TITLE
If TouchOSC /ping enabled, Synergize may crash

### ISSUE THREAD

**Issue by chinenual** [chinenual/synergize#15]
Synergize does not properly filter messages TouchOSC can be configured to send (/accxyx, /z and /ping).  This results in messages such as the following in the log: 

```
2020/11/06 20:20:03 ERROR:  SendToUI failed to decode json response : accxyz 0: unexpected end of JSON input
```
In my testing I was able to crash Synergize by flooding it with /accxyz accelerometer messages; others have reported network related errors (e.g. CONNRESET) which may be related.  See #14 

Until fixed, workaround is to ensure that these are disabled in TouchOSC (`Options->OSC`-> all toggles should be off)

**Comment by chinenual** [comment]
Fixed for version 2.2.1.    Added a blacklist filter to ignore /ping, /accxyz and /z messages from TouchOSC

**Comment by chinenual** [comment]
FWIW, it appears that the crash was caused by /ping messages from TouchOSC -- the accelerometer and touch messages are "harmless" (?), but flood the logs with errors and cause unnecessary traffic between the app and the UI
